### ISSUE TITLE
Blocked by CORS policy

### ISSUE THREAD

**Issue by DheenadhayalanP** [shameemreza/instagram-widget#2]
`jquery.min.js:4 Access to XMLHttpRequest at '/instagram-widget-master/api/?path=%2Fv1%2Fusers%2Fshameemreza_%2F&callback=jQuery112304743327198217022_1550212210362&_=1550212210363' from origin 'null' has been blocked by CORS policy: Cross origin requests are only supported for protocol schemes: http, data, chrome, chrome-extension, https.`

**Comment by shameemreza** [comment]
I guess you are loading the script using either file:// or C:/, which stays true to the error message as they are not http://

So you can either install a webserver in your local PC or upload it somewhere else and use jsonp and change the url to http://example.com/path/to/model

By the way, Firefox will allow you to make AXAJ requests using the file: protocol if the page was loaded using the file: protocol.

Chrome, however, blocks this by default. To enable it, you need to launch Chrome from a command prompt, specifying the --allow-file-access-from-files flag.

**Comment by DheenadhayalanP** [comment]
Thanks for your update, this issue fixed now, but I can't get the expected result, so I close this issue and create a new issue about that bug.
